src/screens/SignedChatScreen/index.tsx

```diff
@@ -52,6 +52,10 @@ const SignedChatScreen = () => {
     goToChatInfoScreen({from: SIGNED});
   };
 
+  const scrollToEnd = () => {
+    flatlistRef.current?.scrollToEnd();
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new function `scrollToEnd` has been added. It seems to be used for scrolling the chat screen to the end. However, there's a typo in the reference to `flatlistRef`. It should be `scrollRef` as defined in the destructuring assignment above.
```diff
-    flatlistRef.current?.scrollToEnd();
+    scrollRef.current?.scrollToEnd();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
this will be deleted since it is not used